dashboard/src/components/OperatorList/OperatorList.tsx

```diff
@@ -8,28 +12,63 @@ export interface IOperatorListProps {
   isFetching: boolean;
   checkOLMInstalled: () => Promise<boolean>;
   isOLMInstalled: boolean;
+  namespace: string;
+  getOperators: (namespace: string) => Promise<void>;
+  operators: IPackageManifest[];
+  error?: Error;
 }
 
 class OperatorList extends React.Component<IOperatorListProps> {
   public componentDidMount() {
     this.props.checkOLMInstalled();
+    this.props.getOperators(this.props.namespace);
   }
 
   public render() {
-    const { isFetching, isOLMInstalled } = this.props;
+    const { isFetching, isOLMInstalled, error } = this.props;
     return (
       <div>
         <PageHeader>
           <h1>Operators</h1>
         </PageHeader>
         <main>
-          <LoadingWrapper loaded={!isFetching}>
-            {isOLMInstalled ? <p>OLM Installed!</p> : <OLMNotFound />}
-          </LoadingWrapper>
+          {error ? (
+            <ErrorSelector
+              error={error}
+              action="list"
+              resource="Operators"
+              namespace={this.props.namespace}
+            />
+          ) : (
+            <LoadingWrapper loaded={!isFetching}>
+              {isOLMInstalled ? this.renderOperators() : <OLMNotFound />}
+            </LoadingWrapper>
+          )}
         </main>
       </div>
     );
   }
+
+  private renderOperators() {
+    return (
+      <CardGrid>
+        {this.props.operators.map(operator => {
+          return (
+            <InfoCard
+              key={operator.metadata.name}
+              link={`/operators/ns/${this.props.namespace}/${operator.metadata.name}`}
+              title={operator.metadata.name}
+              // TODO(andresmgot): Icons are protected by RBAC so we need to use the backend to retrieve those
+              // icon={TBD}
```

Comment:
Won't the RBAC be sufficient when logged in as an admin (ie. for demos) until we also store these in the backend?

Reply:
yes, the problem is that when using `<img src=/link/to/k8s />` the img request doesn't have the authorization header. What we currently do for chart icons is to skip the authorization for logos, we need to do the same for these.

Reply:
The request that gets to the k8s API does have the authorization header if you're using the auth proxy, as cookies get sent with any request. But yes, if you're using the token authentication then it won't. Bummer - the extra complexity to support the token auth is growing.